src/main/java/spring/com/controller/UserController.java

```diff
@@ -0,0 +1,55 @@
+package spring.com.controller;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import spring.com.config.AppConfig;
+import spring.com.dto.UserResponseDto;
+import spring.com.model.User;
+import spring.com.service.UserService;
+import spring.com.service.impl.UserMapperServiceImpl;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    private static final AnnotationConfigApplicationContext context =
+            new AnnotationConfigApplicationContext(AppConfig.class);
+    private static final UserService userService = context.getBean(UserService.class);
+
+    @GetMapping(value = "/{userId}")
+    public UserResponseDto getUser(@PathVariable Long userId) {
+        Optional<User> optionalUser = userService.get(userId);
+        User user = null;
+        if (optionalUser.isPresent()) {
+            user = optionalUser.get();
+            return UserMapperServiceImpl.dtoBoxing(user);
+        }
+        throw new RuntimeException("User " + user + " isn't exist");
```

Comment:
there are a lot of redundant code here
let's remove all checking for Optional and call `get()` or `orElseThrow()` method at the service layer

Reply:
done